src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -317,7 +317,12 @@ public void GetFolderPath_OSX_NonEmptyFolderPaths(Environment.SpecialFolder fold
         public unsafe void GetFolderPath_Windows(Environment.SpecialFolder folder)
         {
             string knownFolder = Environment.GetFolderPath(folder);
-            Assert.NotEmpty(knownFolder);
+
+            if (!PlatformDetection.IsWindowsNanoServer)
+            {
+                // See Nano comment below
+                Assert.NotEmpty(knownFolder);
+            }
```

Comment:
At this point, what is the test actually validating on Nano?  Should the test just be disabled for it with an attribute?